src/sliding-tab/ExNavigationSlidingTab.js

```diff
@@ -33,6 +33,8 @@ type SlidingTabItem = {
 };
 
 type Props = {
+  scrollEnabled?: boolean,
```

Comment:
I think better to name it as `tabScrollEnabled` or something since you're passing it to TabNavigator. Any suggestions for a better name @brentvatne ?